packages/next/build/webpack-config.ts

```diff
@@ -153,6 +153,7 @@ export default async function getBaseWebpackConfig(
       'next/router': 'next/dist/client/router.js',
       'next/config': 'next-server/dist/lib/runtime-config.js',
       'next/dynamic': 'next-server/dist/lib/dynamic.js',
+      'react-dom': path.resolve(path.join(dir, 'node_modules', 'react-dom')),
```

Comment:
I'm hesitant about this change because it covers up broken ecosystem packages, see https://github.com/facebook/create-react-app/issues/675.

Reply:
Can we emit a warning then this alias had to actually be used? Also, we need to include `react` too, right?

Reply:
Turns out this part wasn't actually needed, so I removed it